microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
```

Comment:
Can I propose this for the heading?

Use delegated access with a custom application for Graph PowerShell